capabilities/service.go

```diff
@@ -0,0 +1,109 @@
+// Package capabilities provides an service to find out which capabilities/functions
+// a tenant cluster on the given installation will provide.
+package capabilities
+
+import (
+	"github.com/Masterminds/semver"
+	"github.com/giantswarm/microerror"
+
+	"github.com/giantswarm/gsctl/client"
+)
+
+// Service provides methods to get more details on the installation's
+// and tenant cluster's capabilities.
+type Service struct {
+	// provider identifies the installation's provide,r e. g. 'aws' or 'azure'.
+	provider string
+
+	// client is an API client the service can use to fetch info.
+	clientWrapper *client.Wrapper
```

Comment:
As a last minor nitpick: This could maybe go up so these fields are also sorted alphabetically. :slightly_smiling_face:

Reply:
done